src/runner/tray_icon.cpp

```diff
@@ -83,6 +83,9 @@ void handle_tray_command(HWND window, const WPARAM command_id)
             about_box_shown = false;
         }
         break;
+    case ID_DOCUMENTATION_MENU_COMMAND:
+        ShellExecuteA(NULL, "open", "https://aka.ms/PowerToysOverview", NULL, NULL, SW_SHOWNORMAL);
```

Comment:
Should be `SellExecuteW` instead of `ShellExecuteA`, to be consistent with the rest of the code.

The main issue here is that this may open elevated browser windows from the tray icon when PowerToys is running as an Administrator. Since we are accepting commands from non elevated processes to be able to open settings, I imagine this might be used as an attack vector.

Could you please see if you can include `#include <common/utils/elevation.h>` and try to do `RunNonElevatedEx(L"https://aka.ms/PowerToysOverview", L"");` instead? That should make it so the browser windows start non-elevated.

Reply:
Thanks for this. It is something I was not aware of but very useful to know! I've made the changes.